okhttp/src/test/java/io/grpc/okhttp/OkHttpChannelBuilderTest.java

```diff
@@ -46,11 +48,14 @@
 public class OkHttpChannelBuilderTest {
 
   @Rule public final ExpectedException thrown = ExpectedException.none();
+  @Rule public final GrpcCleanupRule grpcCleanupRule = new GrpcCleanupRule();
 
   @Test
   public void authorityIsReadable() {
     OkHttpChannelBuilder builder = OkHttpChannelBuilder.forAddress("original", 1234);
-    assertEquals("original:1234", builder.build().authority());
+    ManagedChannel channel = builder.build();
+    grpcCleanupRule.register(channel);
```

Comment:
This is designed so it can be inlined:

```java
ManagedChannel channel = grpcCleanupRule.register(builder.build());
// or
assertEquals("original:1234", grpcCleanupRule.register(builder.build()).authority());
```